README.md

```diff
@@ -91,7 +104,7 @@ Environment=NODE_ENV=production
 Type=exec
 Restart=on-failure
 WorkingDirectory=/opt/jellyseerr
-ExecStart=/root/.nvm/versions/node/v18.7.0/bin/node dist/index.js
+ExecStart=node dist/index.js
```

Comment:
Not required if you added the `-g` flag during npm install, node is then added to /bin and accessible directly. 🙂

Reply:
Thats for the node binary. -g flag is used to install dependencies. Very different two things. 

You can add these instructions (just an example. Change wordings so it's clearer 😀 )
Something along the lines of,
- get your node path `which node`
- usually it's `/usr/bin/node`
- assuming it is replace execstart etc 

In systemd service files, it is always a must to use absolute paths. Say the user installed node using NVM. It will not be in `bin`. It will be in their home directory. Meaning a systemd service cannot work with just a `node dist/index.js`.